src/hoi4_world/states/hoi4_states_converter.cpp

```diff
@@ -907,6 +923,11 @@ hoi4::States CreateStates(const vic3::World& source_world,
          const auto [naval_base_location, naval_base_level] =
              DetermineNavalBase(province_set, significant_provinces, coastal_provinces, hoi4_to_vic3_province_mappings);
 
+         int infrastructure = infrastructure_mapper.Map(vic3_state_itr->second.GetInfrastructure());
+
+         const int air_base_level =
+             DetermineAirbaseLevel(civilian_factories + military_factories + dockyards, infrastructure);
```

Comment:
do we want to output total airbase level vs vanilla?

Reply:
I'm not really convinced it rises to that level of concern.